src/lib/recipes.ts

```diff
@@ -422,6 +422,104 @@ const xiphosRecipes: IRecipe[] = [
   ),
 ];
 
+// HH-a "Javelin" half-truck recipes
+const javelinRecipes: IRecipe[] = [
+  // Garage production: 20 x Refined Materials → 1 x HH-a "Javelin"
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 20 }],
+    [{ stuff: Vehicles.Javelin, count: 1 }]
+  ),
+  // Mass Production Factory: 150 x Refined Materials → 3 crates of 3 x HH-a "Javelin" (9 total)
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 150 }],
+    [{ stuff: Vehicles.Javelin, count: 9 }]
+  ),
+];
+
+// HH-b "Hoplite" half-truck recipes
+const hopliteRecipes: IRecipe[] = [
+  // Garage production: 22 x Refined Materials → 1 x HH-b "Hoplite"
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 22 }],
+    [{ stuff: Vehicles.Hoplite, count: 1 }]
+  ),
+  // Mass Production Factory: 165 x Refined Materials → 3 crates of 3 x HH-b "Hoplite" (9 total)
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 165 }],
+    [{ stuff: Vehicles.Hoplite, count: 9 }]
+  ),
+];
+
+// HH-d "Peltast" half-truck recipes
+const peltastRecipes: IRecipe[] = [
```

Comment:
Only one recipe. Requirements for one hoplite:

5 x [Processed Construction Materials](https://foxhole.wiki.gg/wiki/Processed_Construction_Materials) ProcessedConstructionMaterialsIcon.png
5 x [Assembly Materials II](https://foxhole.wiki.gg/wiki/Assembly_Materials_II) AssemblyMaterials2Icon.png
3 x [Assembly Materials IV](https://foxhole.wiki.gg/wiki/Assembly_Materials_IV) AssemblyMaterials4Icon.png
1x [HH-a “Javelin”](https://foxhole.wiki.gg/wiki/HH-a_%E2%80%9CJavelin%E2%80%9D) HalfTrackColVehicleIcon.png